### ISSUE TITLE
EWMB incorrect in dusk and night palettes

### ISSUE THREAD

**Issue by DavidGrant-NIWC** [S-101-Portrayal-subWG/Working-Documents#167]
For consistency with the other color filled symbols the EWMB should use color token OUTLW to stroke the outline. This will keep the outline black in all color palettes.

Affected symbols: `BOYSPH50`, `BOYPIL60`, `BOYSPH60`, `BOYSPR60`

|Dusk|Night|
|:---:|:---:|
|![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/2bba06de-a717-4ac3-a871-43f3030e6f21)|![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/e76440b4-4f74-46f7-bb5d-c1bf4d66dc8c)

**Comment by DavidGrant-NIWC** [comment]
The affected symbols have been updated in the PC to address this issue. Close this issue if the pics below look correct:
|Dusk|Night|
|:---:|:---:|
|![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/491f4fca-ec26-4791-9f6b-c6c8e6e1c018)|![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/808fe4e5-6cd6-40f9-9511-df1ca089e67d)

**Comment by alvarosanuy** [comment]
To be honest I think I prefer to visualise the outline and the position circles of the symbols. In short, I think the symbols you changed should be the standard and all other color filled symbols should be amended to match.
We could track this activity for '2.0.0 or later'.

**To the PsWG team** -  What's your view on this?

**Visible outline examples:**

![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/70189016/290f9aa3-9cf3-41d5-aaf5-da8304ecd858)
![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/70189016/f20bfa48-4cea-41d2-82ed-f93bd0097baa)

**No visible outline examples:**

![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/70189016/95eba503-d4da-4f4b-a5a4-0dd27fcbd9c6)
![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/70189016/9c21fc5b-c8df-482a-b517-339495a7d26d)

**Comment by DavidGrant-NIWC** [comment]
I agree with you and initially was going to change the symbols as you suggest. However, I noticed that all the S-52 color filled symbols use "OUTLW" to outline the color. "OUTLW" stays black in all palettes.

The new color filled symbols are just maintaining consistency with the existing. Changing this would require changing all of the color filled buoys, beacons, and lights. It would also be inconsistent with S-52.

![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/6df24e7e-e4f8-4d2f-b71e-9933bf952091)

**Comment by alvarosanuy** [comment]
**Noted. Your changes to PC 1.3.0 are ok** but, I'll leave this issue open to hear some other points of view or, at least, for discussions at the next PsWG meeting.

For 'Simplified' colored symbols, not 'seeing' the outline is not that bad as they do not have the baseline with the position circle as 'Paper Chart' symbols do. Now that we introduced colored versions of the "Paper Chart' symbols the use of OUTLW in Dusk and Night modes is not ideal.

**Comment by alvarosanuy** [comment]
**Decision at PsWG meeting on 10/7/2024**

- [ ] All Paper Chart navaid symbols included in S-101 need to be reviewed and OUTLW colour replaced with CHBLK.

**Not for PC 2.0.**

**Comment by alvarosanuy** [comment]
**Decision at PsWG meeting on 03/12/2024**

- [ ] **NIWC** to proceed with changes once PC 2.0.0 is endorsed by IHO MS (post 13/12/24).

**Comment by TomRichardson6** [comment]
@DavidGrant-NIWC can you advise if anything further to progress this change?

Noting https://github.com/iho-ohi/S-101_Portrayal-Catalogue/issues/460 can this issue now be closed?

**Comment by DavidGrant-NIWC** [comment]
* This can be closed since we are tracking in the PC issue (noted above).
* There will need to be an issue created to ensure the symbols are updated to the registry once the changes are completed.
* It would also be helpful to understand the process for updating the operational PC; we want to ensure all changes we incorporate into the main PC branch are targeted for inclusion in the next PC release, and that there is a way forward to field those changes (in other words, are there IHO policy issues or does the MS approval process preclude incorporating this change at the current time).
